### ISSUE TITLE
Re-building SoCal Julia

### ISSUE THREAD

**Issue by ChrisRackauckas** [SoCal-Julia/socal-julia.github.io#1]
It seems this is dead. 
- Is there a reason why this has an IRC but not a Gitter?
- What new things are going on in Julia in SoCal?
- Who is moderating this?
- Should the website get an overhaul? It doesn't look nice at all.

**Comment by kshyatt** [comment]
Hi @ChrisRackauckas 

@garrison has moved on to the brighter pastures of Maryland. I am still in SoCal and pretty active and I'd absolutely love to see more community building, but I'm kind of super busy so I haven't done any of the actual work.

The reason why IRC and no Gitter is that when this was active, IRC was far more active than Gitter.

Updating the website (use Jekyll/GH pages if we aren't already?) would be great but I have absolutely no eye for design so I'd have to fob that one off.

New things - I have a dream of hosting a Julia Hackathon either at UCSB or in LA, or both! What are your thoughts?

**Comment by ChrisRackauckas** [comment]
LA would be nice for the Hackathon. I am centered in Irvine, and have been doing a lot with Julia here. We've had workshops (another one coming in the Fall sponsored by the UCI Data Science Initiative), Julia's becoming a part of some courses, etc. I have done blog posts about using Julia on the XSEDE Comet cluster which is based in San Diego. I know they've seen it and have re-tweeted it, but it may be interested to see if XSEDE and Comet would like to do something on getting started with Julia.

If possible, I would be interested in putting some effort to get this going.

**Comment by garrison** [comment]
That would be great, I would be happy to pass on the leadership of the meetup group.

**Comment by garrison** [comment]
> Is there a reason why this has an IRC but not a Gitter?

I prefer methods of chat that are freely implementable and not owned by a single company.  But since I will no longer be involved in this, it is completely up to you guys to decide.

**Comment by ChrisRackauckas** [comment]
@darwindarak picked up the meetup org. I wasn't able to take the organizer position without paying $15/month, so @darwindarak if you had to foot the bill, I'll buy you lunch at one of the meetups!

He's based in UCLA, so maybe he may know of a good central location for some kind of meeting? If not, I've talked to the UCI Data Science Initiative's director and they said we could likely use some of the facilities here.

Talking with @kshyatt, it seems like we may be able to pull together a workshop on parallel / cluster computing with Julia which is associated with XSEDE. Any follow up on that?

---

For reference I like Gitter because if two people are not on at the same time there's a log. Everytime I go to an IRC channel with low activity, I tend to never get a response. We can stay on IRC, but being active there is more difficult.

**Comment by darwindarak** [comment]
I would really like a workshop on cluster computing as well, but I think it might be a little too advanced for a first general meeting after such a long haitus, and not many people have access to a cluster.  Perhaps we can start with something more accessible.  I was thinking along the lines @kshyatt's "Your First Julia Pull Request" and having beginners write tests as a way to learn Julia syntax and features under the guidance of more experienced Julians (also doesn't hurt to get more tests written).  Another good way to kick off the meeting would be for someone to present what's new in v0.5.  I really do think that a parallel computing/cluster workshop would be cool though, and it should definitly be something we explore once we have a better gauge of the attending members.

I'm pretty sure I can get access to some rooms at UCLA as well, so it really comes down to who will be attending and where they are coming from.  Maybe we could even rotate locations between meetings.

Let's setup the polls on meetup.com, since there may be people interested in Julia who might not be used to GitHub.

If this all sounds good, then I'll send emails and polls out to the group and setup a Gitter channel tonight.

**Comment by ChrisRackauckas** [comment]
Yeah, get things going. We can start with something basic. I'll trust your judgement because I haven't done these things before: I would've assumed the "Julia hardcore" would be showing more than Julia newcomers, but you would have more experience. 

I think something like "Your First Julia Project" may be more accessible than "Your First Julia Pull Request". I think a lot more people would be interested in using Julia than contributing to Julia, especially if we're reaching out beyond Github users. 

BTW, could I get SoCal-Julia access to get something going for the website?

**Comment by garrison** [comment]
Great to see some activity.  Thanks for taking over!

Regarding the [location survey](https://www.meetup.com/Southern-California-Julia-Users/polls/1231159/?showResults=1), I am not surprised that most people are from Los Angeles.  In the [initial julia-users thread](https://groups.google.com/forum/#!topic/julia-users/3I36PHniuJY) I was hoping to start a group with people all over SoCal, but the way Meetup suggests groups is based heavily on geographic proximity.  So most people in the group ended up being in LA, even though I was attempting to organize things from Santa Barbara.  In part I was trying to model things on the geographic spread of the [SoCal Python group](http://www.meetup.com/socalpython/), but I think even they don't do anything much north of LA.

Speaking of julia-users, you may wish to announce the reboot there once you've scheduled a next meetup.

**Comment by ChrisRackauckas** [comment]
I will be giving a workshop on Julia as part of the UC Irvine Data Science Initiative on October 25th. If anyone would like to come and help out as a TA I would be happy to put you on the team. This is more UCI-centric, though I don't think there's any restriction that people who sign up have to be affiliated with UCI. I'm trying to make this a quarterly thing, and I'll get the site up and add this ASAP.

But, how's our next meet-up going? It looks like UCLA would be a good location for it. Can you get a room? Let's get a room, a title, and a general schedule down.

**Comment by darwindarak** [comment]
I'm in NorCal at the moment, so I'll schedule the room when I'm back in LA on Monday.  Iwhat do you think about setting the meeting during the last week of September? 'll send out an email tomorrow to poll for the best time of the week to meet and maybe a call for speakers.

**Comment by ChrisRackauckas** [comment]
That sounds good.

**Comment by ChrisRackauckas** [comment]
Did you ever schedule a meeting? Should we advertise to julia-users?

**Comment by darwindarak** [comment]
Last time I checked it looks like Friday is the best day for everyone. Do you mind checking if that's still the case? I'm wifi-less at the moment and internet is spotty on my phone. If Friday works then I'll go ahead and find a room from 6 to 8. Can you make announcement on Julia-users? I'll update it with the room number once I get a confirmation.

**Comment by ChrisRackauckas** [comment]
(I assume that's 6PM-8PM, correct?)

**Comment by darwindarak** [comment]
Yes

**Comment by ChrisRackauckas** [comment]
I won't be able to make the Friday meeting. I am required to go to an event that will last until 6:30. I wish I knew earlier, and sorry for the late cancel. Let me know if you need any help. The Julia-users announcement has been up, though it looks like attendance is small.

**Comment by darwindarak** [comment]
So that didn't work out so well. It looks like most people can't make it. Let's figure out another time.

**Comment by kshyatt** [comment]
Thoughts on a post-Thanksgiving meetup?

**Comment by ChrisRackauckas** [comment]
Sounds interesting. As @darwindarak mentioned, maybe we should blend it with a workshop or something? I'd be interested.

**Comment by ChrisRackauckas** [comment]
We had one great meeting at Cal Tech by @simonbyrne. Now I'm planning another at UCI. I wonder if anyone would like to give a talk on something? @lstagner ? @YingboMa ? I have some stuff planned, but would like others to be on board as well.

**Comment by kshyatt** [comment]
I could probably give a talk!

**Comment by ChrisRackauckas** [comment]
@kshyatt @simonbyrne let's try to pick a day. Just throwing one out there: February 7th?

**Comment by ChrisRackauckas** [comment]
February 7th at 7. I'll create the meetup once we have titles.

**Comment by lstagner** [comment]
Is there a poster I could use? I was going to hang up flyers on the bulletin board in FRH

**Comment by ChrisRackauckas** [comment]
I don't have a poster. Anyone have a design?

**Comment by lstagner** [comment]
I've made a quick poster
[julia-users-poster.pdf](https://github.com/SoCal-Julia/socal-julia.github.io/files/1696925/julia-users-poster.pdf)

**Comment by lstagner** [comment]
If there are no objections I will print up a few and put them in FRH and Rowland Hall

**Comment by ChrisRackauckas** [comment]
LGTM
